ui/datasets/semantic-ui/js/datasets/forms/actions.js

```diff
@@ -0,0 +1,41 @@
+import { _saveDraft } from "@js/oarepo_ui/forms/state/deposit/actions";
+import { i18next } from "@translations/i18next";
+import {
+  DRAFT_HAS_VALIDATION_ERRORS,
+  DRAFT_SAVE_FAILED,
+  DRAFT_SAVE_STARTED,
+  DRAFT_SAVE_SUCCEEDED,
+} from "@js/invenio_rdm_records/src/deposit/state/types";
+
+export const save = (
+  draft,
+  {
+    successMessage = i18next.t("Draft saved successfully."),
```

Comment:
There will be no draft :) We directly save changes to record here

Reply:
You're right, lets keep it, just make sure that actions call the right API service endpoints on dispatch